worker/worker.go

```diff
@@ -1144,6 +1146,36 @@ func (w *worker) idHandlerGET(jc jape.Context) {
 	jc.Encode(w.id)
 }
 
+func (w *worker) handleAlertsWebhooksPOST(jc jape.Context) {
+	var req webhooks.WebHookRegisterRequest
```

Comment:
all of our requests and response types are in the `api` package, I think we should move this there as well